### PR TITLE
feat: replacing custom coverage QC scripts with "alfred qc" (#481)

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

**Comment by holtgrewe** [comment]
@Nicolai-vKuegelgen This will need:

- `rm -rf {work,output}/*/report/cov_qc`

In `ngs_mapping` to ensure 100% reprocessing.

File: snappy_wrappers/wrappers/varfish_annotator/bam_qc/wrapper.py [hunk 1]
```diff
@@ -32,6 +36,48 @@
         print(f"{library_name} {identifier}", file=json_tmpf)
         json_tmpf.flush()
 
+    # Extract the coverage data from the alfred qc files.
+    for alfred_qc in snakemake.input.alfred_qc:
+        with gzip.open(alfred_qc, "rt") as inputf:
+            data = json.load(inputf)
```
Comment (Nicolai-vKuegelgen) [inline snappy_wrappers/wrappers/varfish_annotator/bam_qc/wrapper.py:42]: This looks like only the last input file will be read into `data`, is that really intended?

 Or should the next for-loop be indented one level further?

**Review by Nicolai-vKuegelgen — COMMENTED** [review]
Only one thing I am not sure about, rest looks good.

File: snappy_wrappers/wrappers/varfish_annotator/bam_qc/wrapper.py (hunk 1 above)
Comment (holtgrewe) [reply snappy_wrappers/wrappers/varfish_annotator/bam_qc/wrapper.py:43]: True

**Pull request merged**